dotnet/src/Skills/Skills.MsGraph/CalendarSkill.cs

```diff
@@ -119,4 +134,53 @@ public async Task AddEventAsync(string subject, SKContext context)
         this._logger.LogInformation("Adding calendar event '{0}'", calendarEvent.Subject);
         await this._connector.AddEventAsync(calendarEvent).ConfigureAwait(false);
     }
+
+    /// <summary>
+    /// Get calendar events with specified optional clauses used to query for messages.
+    /// </summary>
+    [SKFunction("Get calendar events.")]
+    [SKFunctionContextParameter(Name = Parameters.MaxResults, Description = "Optional limit of the number of events to retrieve.")]
+    [SKFunctionContextParameter(Name = Parameters.Skip, Description = "Optional number of events to skip before retrieving results.")]
+    public async Task<string> GetCalendarEventsAsync(SKContext context)
+    {
+        this._logger.LogInformation("Getting calendar events with query options top: '{0}', skip:'{1}'.",
+            context.Variables.Get(Parameters.MaxResults, out string maxResultsString),
```

Comment:
If not specified, will this return ALL calendar events in the future? That could easily be in the hundreds, no?

Reply:
In the graph SDK case, it paginates, but since this is not graph specific, I'll add default values of "10" max and "0" skip.